Makefile

```diff
@@ -234,7 +234,7 @@ docker-build-debug:
 # TODO: Push this to the Cosmos Dockerhub so we don't have to keep building it
 # in CI.
 docker-build-hermes:
-	@cd tests/e2e/docker; docker build -t ghcr.io/cosmos/hermes-e2e:1.4.1 -f hermes.Dockerfile .
+	@cd tests/e2e/docker; docker build -t ghcr.io/cosmos/hermes-e2e:1.0.0 -f hermes.Dockerfile .
```

Comment:
Could you give a sentence of context for the change in the minor version numbers? LGTM apart from that